packages/web-components/src/components/va-table/va-table.scss

```diff
@@ -0,0 +1,44 @@
+@use 'usa-table/src/styles/usa-table';
```

Comment:
Do you need to also @forward the USWDS settings?

Reply:
@jamigibbs good catch - done!